lib/window.ts

```diff
@@ -8,13 +8,50 @@ export default function createWindow(currentWindow: Window, channel: Channel): W
   const autoUpdateHeight = () => {
     self.updateHeight()
   }
-  const observer = new MutationObserver(autoUpdateHeight)
+
+  const heightObserverCallback = (mutations: Array<MutationRecord>) => {
+    checkAbsolutePositionedElems(mutations)
+    autoUpdateHeight()
+  }
+
+  const observer = new MutationObserver(heightObserverCallback)
   let oldHeight: number
   let isAutoResizing = false
+  const absolutePositionedElems: Set<Node> = new Set()
```

Comment:
This should be `Set<Element>`, right?
We don't really want to have any other node types in here